modules/command/publish/insert-assertion-command.js

```diff
@@ -15,14 +15,21 @@ class InsertAssertionCommand extends Command {
      * @param command
      */
     async execute(command) {
-        const { documentPath, handlerId } = command.data;
+        const { documentPath, handlerId, operationId } = command.data;
```

Comment:
Perhaps we can make only one id that servers as handler and operation id in future